packages/connection-form/src/components/advanced-options-tabs/csfle-tab/kms-provider-fields.tsx

```diff
@@ -71,7 +71,7 @@ function KMSProviderFieldsForm<KMSProvider extends keyof KMSProviders>({
                   handleFieldChanged(name, value);
                 }}
                 name={name}
-                data-testid={name}
+                data-testid={`csfle-kms-${name}`}
```

Comment:
This is so much better! it would break the other tests i just wrote, do you want me to do this change in the other PR so you'd only need to rebase?

Reply:
Feel free to change it in your PR!